### DISCUSSION TITLE
[Feature] Custom TLS certificate

### DISCUSSION THREAD

**Discussion by LordArrin** [immich-app/immich#12821] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Please consider adding support for custom tls certificates, like it's done on another projects, like qbittorrent-web-ui and Jellyfin. It can be much easier solution then proxies or other crap.
![Screenshot_20240920-224258_Jellyfin](https://github.com/user-attachments/assets/cfa197c9-bd2a-456f-9fe3-5e080f098988)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by LordArrin** [comment]
I've seen [that](https://github.com/immich-app/immich/discussions/1611) discussion, but not sure it is the same. My English isn't that good

**Comment by bo0tzz** [comment]
We have no plans to do this, a reverse proxy is perfectly well suited to the job.
